guild.php

```diff
@@ -0,0 +1,150 @@
+        $guildId,
+        $players_group_and_order);
+
+    $result = $db->query($query);
+
+    if (!$result)
+        die("Error querying: " . $query);
+
+    $position = 0;
+    $prev_score = 0;
+
+    while (($row = $result->fetch_array()) != null)
+    {
+        if ($prev_score != $row['count'])
+            $position++;
+
+        $player_name = sprintf("<span style=\"color: %s; \"><strong>%s</strong></a>",
+            getPlayerColorByRace($row['character_race']),
+            $row['character_name']);
+
+        printf("
+            <tr>
+                <td>%d</td>
+                <td>%s</td>
+                <td style=\"min-width: 46px; padding-left: 0; padding-right: 0;\"><img src=\"img/class/%d.gif\"> <img src=\"img/race/%d-%d.gif\"></td>
+                <td>%s</td>
+                <td>%d</td>
+                <td><strong>%d</strong></td>
+            </tr>",
+            $position,
+            $player_name,
+            $row['character_class'],
+            $row['character_race'],
+            $row['character_gender'],
+            $row['character_level'],
+            $row['character_totalKills'],
+            $row['count']
+        );
+
+        $prev_score = $row['count'];
+    }
+}
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="PvPstats, see who is winning!">
+    <meta name="author" content="ShinDarth">
+
+    <title><?= $server_name ?> PvPstats</title>
+
+    <link href="css/bootstrap-cyborg.min.css" rel="stylesheet">
+    <link href="css/style.css" rel="stylesheet">
+    <link href="css/top100-style.css" rel="stylesheet">
+  </head>
+
+  <body>
+
+    <?php require_once("navbar.php"); ?>
+
+    <div class="container">
+
+      <div class="row text-center">
+        <div class="col-sm-12 col-lg-10 col-lg-offset-1" style="padding: 0 10px;">
+          <div class="main-title" style="margin-top: 30px;">
+            <p class="text-center h4">Guild <span style="color: orange;">&lt;<?=$guildName?>&gt;</span> of <?=$server_name?></p>
+          </div>
+          <p class="text-center" style="margin-top: 5px">The sum of the victories of all guild members determines the guild score in the <a href="top100.php">Top100</a>.</p>
+          <div class="top100 table-responsive">
+            <table class="table table-striped">
+              <thead>
+                <tr>
+                  <th class="text-center">#</th>
+                  <th class="text-center">Character</th>
+                  <th class="text-center">&#9679;</th>
+                  <th class="text-center">Level</th>
+                  <th class="text-center">Kills</th>
+                  <th class="text-center">Victories</th>
+                </tr>
+              </thead>
+              <tbody>
+                <?php getGuildPlayers(3); ?>
```

Comment:
?

Reply:
is it not better to return the characters and print them using `<?= getGuildPlayers(3) ?>`

Reply:
yes, also I should remove the `3`